sale_order_invoicing_grouping_criteria/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Sales order invoicing grouping criteria',
-    'version': '11.0.1.0.0',
+    'version': '11.0.1.1.0',
```

Comment:
Please don't change the module version anymore. The bot does it and avoids conflicts.